kills counted (output not working yet)

jsstrickler opened the pull request.

jsstrickler committed:
kills counted (output not working yet)

Nilsosi commented on server.js:
```diff
@@ -36,7 +36,7 @@ function getRandomPosition() {
   };
 }
 
-function explodeBomb(bomb) {
```
Sollte der "placer" nicht teil der bombe sein und nicht seperat übergeben werden?

Nilsosi commented on server.js:
```diff
@@ -36,7 +36,7 @@ function getRandomPosition() {
 }
 
-function explodeBomb(bomb) {
+function explodeBomb(bomb, placer) {
```
Sollte der "placer" nicht teil der bombe sein und nicht seperat übergeben werden?

jsstrickler committed:
fixed requested changes & killcounter working

Not sure if the urls are right after changing to localhost and back. Should be checked for any mistakes before merging.

Nilsosi committed:
reset url

Nilsosi committed:
reset url
